Library/Formula/mplayer.rb

```diff
@@ -67,7 +67,7 @@ def install
 +++ b/configure
 @@ -1517,8 +1517,6 @@ if test -e ffmpeg/mp_auto_pull ; then
  fi
- 
+
```

Comment:
Careful of tweaking patch spacing like this. Has the potential to cause fun little breakage depending on user configuration.